air-script/tests/list_comprehension/list_comprehension.rs

```diff
@@ -0,0 +1,96 @@
+use winter_air::{Air, AirContext, Assertion, AuxTraceRandElements, EvaluationFrame, ProofOptions as WinterProofOptions, TransitionConstraintDegree, TraceInfo};
+use winter_math::fields::f64::BaseElement as Felt;
+use winter_math::{ExtensionOf, FieldElement};
+use winter_utils::collections::Vec;
+use winter_utils::{ByteWriter, Serializable};
```

Comment:
nit: these could be condensed into 2 lines

Reply:
I'm not sure how to do that. The import function takes a module path and type. So, currently we do:
```
scope.import("winter_utils::collections", "Vec")
```
If I move `collections` to the type argument:
```
scope.import("winter_utils",  "collections::Vec")
```
this generates output:
```
use winter_utils::{collections, ByteWriter, Serializable};
```
This is what I had done earlier to combine the imports before Bobbin pointed out the problem.

Looking at the import constructor it just joins the two strings:
```
pub fn new(path: &str, ty: &str) -> Self {
        Import {
            line: format!("{}::{}", path, ty),
            vis: None,
        }
    }
```
So, will have to look into it more. If you have any ideas on how to achieve this, please let me know otherwise I'll dig into this more later.

Reply:
Ah yeah, makes sense. It's not worth any time before the next version, so let's just leave it.